script.js

```diff
@@ -1,4 +1,68 @@
+// initialise first mode asking input
+var mode = 'need input';
+
+// two dice array for ramdom dice roll
+var diceOne = [];
+var diceTwo = [];
+
+// player 1 and 2 number
+var player1Dice = [];
+var player2Dice = [];
```

Comment:
1. These var names are misleading. The names and data structures imply that they will store a list of individual dice rolls, but in fact they store a single value: the concatenated dice roll numbers. 
2. Why do we use arrays here?
3. Can we name the variables something more precise, e.g. `p1FinalNumber` or `p1CombinedNumber`?